### ISSUE TITLE
security.txt has expired on 2024-03-14

### ISSUE THREAD

**Issue by oh2fih** [securitytxt/securitytxt.org#97]
The key used to sign the current [`security.txt`](https://github.com/securitytxt/securitytxt.org/blob/master/.well-known/security.txt) that `Expires: 2024-03-14T00:00:00.000Z` has expired on `Wed 23 Aug 2023`.

```
gpg: Signature made Wed 01 Feb 2023 06:20:57 PM EET
gpg:                using EDDSA key AC3F6904768283545A5283ABB7ACAF980A48DAA7
gpg: Note: signature key B7ACAF980A48DAA7 expired Wed 23 Aug 2023 12:34:19 PM EEST
gpg: Note: signature key B7ACAF980A48DAA7 expired Wed 23 Aug 2023 12:34:19 PM EEST
gpg: Note: signature key B7ACAF980A48DAA7 expired Wed 23 Aug 2023 12:34:19 PM EEST
gpg: using pgp trust model
gpg: key 50C62EAE952C56BF: accepted as trusted key
gpg: key 3B04D423957EFE0F: accepted as trusted key
gpg: Good signature from "Ed Foudil <contact@edoverflow.com>" [expired]
gpg: Note: signature key B7ACAF980A48DAA7 expired Wed 23 Aug 2023 12:34:19 PM EEST
gpg: Note: This key has expired!
```

The [RFC 9116, 2.5.5](https://www.rfc-editor.org/rfc/rfc9116#section-2.5.5) does not have a recommendation against signing a `security.txt` with a key that expires before the signed file becomes stale. In my opinion it should have had, as it does not make much sense to trust the information if you do not trust the signature. (For this reason, my [`securitytxt-signer.sh`](https://github.com/oh2fih/securitytxt-tools/tree/master?tab=readme-ov-file#securitytxt-signersh) [L115-134](https://github.com/oh2fih/securitytxt-tools/blob/33a34a45ca3abf50c0893ea9174117f3687d88a5/securitytxt-signer.sh#L115-L134) never accepts an expiry date that is further in the future.)

**Comment by oh2fih** [comment]
Well, today the [`security.txt`](https://github.com/securitytxt/securitytxt.org/blob/67221f754da0555fd353a52f08e0aae95e1f8e1c/.well-known/security.txt) has expired, too.

```
Expires: 2024-03-14T00:00:00.000Z
```

**Comment by oh2fih** [comment]
Fixed in 46425fea5726547c880cee3b2043f55599b2c5dd.
